uwrt_arm_cartesian_controller/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
```

Comment:
since this is an arm specific package, it should be put in that meta pkg mybe something like uwrt_mars_rover_arm/arm_controllers/cartesian_controller (with the same namespacing converntion that other packages are using)